generators/generator-constants.mts

```diff
@@ -25,20 +25,20 @@ export const JAVA_COMPATIBLE_VERSIONS = ['17', '18', '19'];
 export const ADD_SPRING_MILESTONE_REPOSITORY = false;
 
 // Version of Node, NPM
-export const NODE_VERSION = '18.16.0';
-export const OPENAPI_GENERATOR_CLI_VERSION = '2.5.1';
+export const NODE_VERSION = '18.16.1';
+export const OPENAPI_GENERATOR_CLI_VERSION = '2.6.0';
 
 export const javaDependencies: Record<string, string> = {
   /**
    * spring-boot version should match the one managed by https://mvnrepository.com/artifact/tech.jhipster/jhipster-dependencies/JHIPSTER_DEPENDENCIES_VERSION
    */
-  'spring-boot': '3.1.0',
+  'spring-boot': '3.1.1',
   /*
    * hibernate version should match the one managed by https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies/SPRING_BOOT_VERSION
    * Required due to hibernate-jpamodelgen annotation processor.
    */
   // TODO update to spring-boot provided version. Refer to https://github.com/jhipster/generator-jhipster/issues/22579
-  hibernate: '6.1.7.Final',
+  hibernate: '6.1.7.Final', // Should be 6.2.5.Final
```

Comment:
Is there a bug in Hibernate 6.2 to track this or is it something we need to change?

Reply:
Related with https://github.com/jhipster/generator-jhipster/issues/22579

Reply:
May I ask if updating to 6.2.5 still failing?